# Single scan from a Bus CAN

int8 senal


# Señal 1 - ELECTRONIC TRANSMISSION CONTROLLER1
string opcion_etc1		#string
float64 revoluciones_etc1	#double
float64 porcentaje_etc1		#double
float64 revoluciones2_etc1	#double


# Señal 2 - ELECTRONIC ENGINE CONTROLLER1
float64 revoluciones_eec1	#double
float64 porcentaje_eec1		#double (entero)


# Señal 3 - ELECTRONIC TRANSMISSION CONTROLLER2
float64 marcha			#double (entero)
float64 marcha_2		#double
float64 marcha_3		#double (entero)


# Señal 4 - ELECTRONIC BRAKE CONTROLLER1
string opcion_ebc1		#string
float64 porcentaje_ebc1		#double


# Señal 5 - CRUISE CONTROL VEHICULE SPEED
string opcion_ccvs		#string
float64 km_h_ccvs		#double
string opcion2_ccvs		#string
string opcion3_ccvs		#string
string opcion4_ccvs		#string
string pto_state_ccvs		#string


# Señal 6 - AUXILIARY STATE
string opcion_aux		#string
string opcion2_aux		#string
string opcion3_aux		#string


# Señal 7 - ELECTRONIC ENGINE CONTROLLER2
string opcion_eec2		#string
string opcion2_eec2		#string
float64 porcentaje_eec2		#double
float64 porcentaje2_eec2	#double


# Señal 8 - ENGINE TEMPERATURE
float64 grados_et		#double (entero)
float64 grados2_et		#double (entero)
float64 grados3_et		#double


# Señal 9 - ENGINE FLUID
float64 bares_ef		#double


# Señal 10 - SUPPLY PRESSURE 
float64 bares_sp		#double
float64 bares2_sp		#double
float64 bares3_sp		#double
float64 bares4_sp		#double
float64 bares5_sp		#double
float64 bares6_sp		#double


# Señal 11 - AMBIENT CONDITIONS
float64 bares_ac		#double
float64 grados_ac		#double


# Señal 12 - VEHICULE DISTANCE HIGH RESOLUTION
float64 kilometros_vdhr		#double


# Señal 13 - VEHICULE WEIGHT
string axle			#string
float64 kilogramos_vheacs	#double


# Señal 14 - ENGINE HOURS
float64 horas			#double


# Señal 15 - TACOGRAPH
string driverWorking1		#string
string driverWorking2		#string
string opcion_t			#string
string driverTime1		#string
string opcion2_t		#string
string opcion3_t		#string
string driverTime2		#string
string opcion4_t		#string
string opcion5_t		#string
string opcion6_t		#string
string opcion7_t		#string
float64 km_h_t			#double


# Señal 16 - ELECTRONIC RETARDER CONTROLLER EXHAUST
float64 porcentaje_erc		#double (entero)


# Señal 17 - AUX STAT
string opcion_auxstat		#string
string opcion2_auxstat		#string


# Señal 18 - FUEL ECONOMY
float64 litros			#double
float64 km_l			#double


# Señal 19 - ELECTRONIC TRANSMISSION CONTROLLER3 
string opcion_etc3		#string
string opcion2_etc3		#string
string pto1state		#string
string pto2state		#string
string nmvstate			#string


